### ISSUE TITLE
Wrong string for new game save in some languages

### ISSUE THREAD

**Issue by Bruno3589** [elishacloud/Silent-Hill-2-Enhancements#553]
I noticed another minor bug while testing the fix for issue #544, however it's not related to that problem, hence why I'm opening a new issue.
I stumbled on it when I created a "new game" save, I mean the very first save you do when there isn't any previous one.

The thing is that the Italian string states it needs 32 _blocks_ for a new game save:
![immagine](https://user-images.githubusercontent.com/12817088/177423235-49c009ad-8e5d-4c61-90f2-6486320b3f75.png)

Which is a little weird since the string for subsequent saves correctly states it needs 8 _KB_:
![immagine](https://user-images.githubusercontent.com/12817088/177424348-9fe07bdc-d913-46ed-8b23-2bab09e1b7a9.png)

So I dug a little deeper and tested all languages: it turns out English and Spanish correctly state they need 32 KB for a new game save, while German, French and Italian ask for 32 blocks.

After that I checked the Italian language file and it turns out both strings are present in the translation, the one with "KB" and the one with "blocks". If I understand correctly "blocks" is supposed to be for the Xbox save, while "KB" is for the PC save.

So it seems the game picks the wrong string for German, Italian and French.
If I remember correctly something similar happened previously and it was fixed by pointing to the correct string, I suppose you need some sort of address for that, but I'm not sure if I can find it in the language file.

Again, this is also a very minor issue, and it would pass unnoticed most of the times since it only occurs when there are no other saves.
I suppose it's worth fixing if it's not too complicated, but there's no rush :)

**Comment by elishacloud** [comment]
As I understand it, a new game save requires 32 KBs, for the very first save.  However, a new save after that is only 8 KBs.

I think what is listed as "32 blocks" should be "32 KB".

**Comment by Polymega** [comment]
You'll need to edit lines 288, 290, 338, and 341:
![image](https://user-images.githubusercontent.com/27395260/177428250-ed7dea35-5053-46b8-b2d4-fbbff838fac1.png)
![image](https://user-images.githubusercontent.com/27395260/177428306-08ab295f-c6f8-46bd-9542-640ff12a2c6a.png)

Looks like you edited it for the other lines but not those ones:

![image](https://user-images.githubusercontent.com/27395260/177428347-56edae52-8185-402e-b0ea-93f5cb1bab3a.png)
![image](https://user-images.githubusercontent.com/27395260/177428364-b0fbe4ed-ad2e-434a-8d84-1d5c0ad58950.png)

Same issue is likely happening for French and German. I noticed this same thing for the English .mes file. There were multiple lines for these save strings, but the game only uses particular ones.

**Comment by Bruno3589** [comment]
So you're saying it's not picking the wrong string, but the string in those languages if wrong... That could be, the strings for the "new game" save and the subsequent ones is very similar so it's possible I got confused. I'll double-check by comparing string positions between Italian and English.

**Comment by Polymega** [comment]
In that .mes file, there are several instances of these strings. I believe you correctly edited the first four, but didn't edit the last four. Chances are you changed the first batch you saw and thought it was good. I accidentally did this to when I was in the English version of this file. You'll just need to translate lines 288, 290, 338, and 341 to your liking.

**Comment by Bruno3589** [comment]
So I compared Italian and English m_card_x_msg:
- The first two string you mentioned (288 and 290) also state "block" in the English file, so I assume they're not used on PC.
- The other two actually stated "block" in italian and "KB" in english, so those were the wrong ones.

Since you mentioned there were several instances of the string I searched and corrected them all... AND I also found out why I was confused: the strings for the additional saves were sometimes translated as "new save"! 🤦 
I also slightly rephrased the strings and now they fit on one line even in 4:3 aspect ratio.

Here's the updated m_card_x_msg_i.mes and m_card_msg_i.mes:
